index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="author" content="Emmanuel Hagan" />
+  <title>Hagan's Portifio</title>
+  <!-- Stylesheet -->
+  <link rel="stylesheet" href="assets/css/style.css">
+  <link rel="stylesheet" href="assets/css/layout.css">
+</head>
+
+<body>
+
+  <header class="toolbar">
+
+       <a href="#" class="logo-heading">Hagan</a>
+      <img class="icon" src="assets/image/icon-menu.png" alt="Icon menu">
+ 
+  </header>
+  <!-- End Header  -->
+
+  <!-- Section  -->
+  <section class="headline">
+    <h1 class="intro">I’m Emmanuel Hagan Glad to see you!</h1>
+    <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some
+      of my work and
+      experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+
+    <a class="connect">LET'S CONNECT <br> </a>
+    <!---Social Link-->
+
+    <ul class="social-links">
+      <li>
+        <a href=""><img src="assets/image/social_icon/medium.png" class="social-icon" alt="Medium"></a>
+      </li>
+      <li>
+        <a href=""><img src="assets/image/social_icon/angellist.png" class="social-icon" alt="angellist"></a>
+      </li>
+      <li>
+        <a href=""><img src="assets/image/social_icon/twitter.png" class="social-icon" alt="twitter"></a>
+      </li>
+      <li>
+        <a href=""><img src="assets/image/social_icon/linkedin.png" class="social-icon" alt="linkedin"></a>
+      </li>
+      <li>
+        <a href=""><img src="assets/image/social_icon/github.png" class="social-icon" alt="github"></a>
+      </li>
+
+      <!---End Social Link-->
+    </ul>
+
+    <img class="ellipse3" src="assets/image/ellipse/Ellipse3.png"  alt="one">
+    <img class="ellipse11" src="assets/image/ellipse/Ellipse11.png"  alt="one2">
+    <img class="ellipse12" src="assets/image/ellipse/Ellipse12.png"  alt="one3">
+    <img class="ellipse13" src="assets/image/ellipse/Ellipse13.png"  alt="one4">
+    <img class="ellipse14" src="assets/image/ellipse/Ellipse14.png"  alt="one6">
+
+    <img class="rectangle4" src="assets/image/ellipse/Rectangle4.png" alt="one">
+    <img class="rectangle5" src="assets/image/ellipse/Rectangle5.png"  alt="one6">
+    <img class="rectangle6" src="assets/image/ellipse/Rectangle6.png" alt="one7">
+
```

Comment:
- [ ] Please remove these shape images because the background image of that section suppose to be added in `style.css` file as a `background-image` you can learn more about it [here](https://www.w3schools.com/cssref/pr_background-image.asp) and also you can use this [link ](https://www.w3schools.com/cssref/pr_background-position.asp)to position it right.

[Hint]: You can export the whole background of this section as one PNG or SVG photo from Figma.